src/NBInclude.jl

```diff
@@ -56,10 +56,20 @@ If the cell has no number (e.g. if it hasn't been evaluated yet), then
 it is assigned a number `+N` for the `N`-th nonempty cell.  If `renumber`
 is set to `true`, then the cell numbers saved in the notebook are ignored
 and each cell is assigned a consecutive number `N`.
+
+`counters` and `regex` can be used to limit the execution of notebook cells.
+Only cells for which `counter ∈ counters` holds or the cell text matches `regex`
```

Comment:
Isn't it "and" not "or"?  i.e. both conditions need to hold.   Or should we change the code to `||`?